extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/config/AuthFieldType.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.devui.runtime.config;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public enum AuthFieldType {
+
+    PERMISSION_ENABLED(AuthConfigType.PERMISSION, "enabled"),
+    PERMISSION_POLICY(AuthConfigType.PERMISSION, "policy"),
+    PERMISSION_METHODS(AuthConfigType.PERMISSION, "methods"),
+    PERMISSION_PATHS(AuthConfigType.PERMISSION, "paths"),
+    PERMISSION_AUTH_MECHANISM(AuthConfigType.PERMISSION, "auth-mechanism"),
+    POLICY_ROLES_ALLOWED(AuthConfigType.POLICY, "roles-allowed"),
+    POLICY_PERMISSIONS(AuthConfigType.POLICY, "permissions"),
```

Comment:
There is new property `roles` on the role-based permissions that is missing here. 

This sort of consistency is one of reasons why I hoped only config form editor was enhanced instead of new `AuthProcessor`. We don't have tests for UI as it's difficult and they are generally slower (we Quarkus QE have some and it's generally slow to download and start playwright comparing to `QuarkusTest` without container) and there is no way to ensure someone will not forget to update `AuthFieldType`.